sos/report/plugins/nvidia.py

```diff
@@ -23,13 +23,22 @@ def setup(self):
             '--list-gpus',
             '-q -d PERFORMANCE',
             '-q -d SUPPORTED_CLOCKS',
-            '-q -d PAGE_RETIREMENT'
+            '-q -d PAGE_RETIREMENT',
+            '-q',
+            '-q -d ECC',
+            'nvlink -s',
+            'nvlink -e'
         ]
 
         self.add_cmd_output(["nvidia-smi %s" % cmd for cmd in subcmds])
 
         query = ('gpu_name,gpu_bus_id,vbios_version,temperature.gpu,'
-                 'utilization.gpu,memory.total,memory.free,memory.used')
+                 'utilization.gpu,memory.total,memory.free,memory.used'
+                 'clocks.applications.graphics,clocks.applications.memory')
+        querypages = ('timestamp,gpu_bus_id,gpu_serial,gpu_uuid,'
+                       'retired_pages.address,retired_pages.cause')
         self.add_cmd_output("nvidia-smi --query-gpu=%s --format=csv" % query)
+        self.add_cmd_output("nvidia-smi --query-gpu=%s --format=csv" % query)
```

Comment:
Just saw this duplication, was this intentional?

Reply:
This was not intentional..copy paste error..Thanks

Reply:
Thanks for the review, I have updated all your comments and sent new version